torchok/constructor/constructor.py

```diff
@@ -329,10 +329,8 @@ def _prepare_transforms_recursively(transforms: ListConfig) -> List[Union[A.Comp
             transform_name = transform_info.name
             transform_params = transform_info.get('params', dict())
 
-            if transform_name in ['Compose', 'OneOf', 'SomeOf', 'PerChannel', 'Sequential']:
+            if 'transforms' in transform_params:
```

Comment:
It's hard to understand the logic behind this. Why did you change the original implementation?

Reply:
Because before, we were limited to adding new container augmentations. The common thing for all container transforms is the 'transforms' parameter requirement. So, now we can add custom container augmentations

Reply:
Please add a comment about it here

Reply:
Added